bc_obps/reporting/service/report_sign_off_service.py

```diff
@@ -1,14 +1,50 @@
 import datetime
 from django.db import transaction
 from reporting.models.report_sign_off import ReportSignOff
-from reporting.schema.report_sign_off import ReportSignOffAcknowledgements, ReportSignOffIn
 from django.core.exceptions import ValidationError
 
 
+class ReportSignOffAcknowledgements:
+    def __init__(
+        self,
+        acknowledgement_of_review: bool,
+        acknowledgement_of_records: bool,
+        acknowledgement_of_information: bool | None,
+        acknowledgement_of_possible_costs: bool,
+        acknowledgement_of_new_version: bool | None,
+    ):
+        self.acknowledgement_of_review = acknowledgement_of_review
+        self.acknowledgement_of_records = acknowledgement_of_records
+        self.acknowledgement_of_information = acknowledgement_of_information
+        self.acknowledgement_of_possible_costs = acknowledgement_of_possible_costs
+        self.acknowledgement_of_new_version = acknowledgement_of_new_version
```

Comment:
Looks like this would do the exact same thing! I'd be up for experimenting this as a pattern for our service communication across our app

```suggestion
@dataclass
class ReportSignOffAcknowledgements:
        acknowledgement_of_review: bool,
        acknowledgement_of_records: bool,
        acknowledgement_of_information: bool | None,
        acknowledgement_of_possible_costs: bool,
        acknowledgement_of_new_version: bool | None,
```